### ISSUE TITLE
Nav bar links - consider underline upon hover

### ISSUE THREAD

**Issue by kerchner** [gwu-libraries/sfm-ui#1036]
Although lack of underlines was determined not to be an accessibility problem, we might consider upon-hover to improve usability.
